### PR TITLE
makes engi thrusters more hearable - x3 the sound

### LINKED ISSUE #2706: Engine thrusters issues.
The sound they create is only hear able when you're standing right outside their windows.
They only create 'puffs' and sounds when they start moving, not while they're moving?

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
makes the sound the engi thrusters make x3 more load

## Details
closes #2706

### CODE REVIEW

**Comment by SpadesSlick** [comment]
How exactly loud it's in the game?

**Comment by Trilbyspaceclone** [comment]
3x louder form not really hearable. Seeings how you can **bearly** hear it well next to them you should not be able to hear them past the engi fuel cans ect ect

**Comment by ACCount12** [comment]
Have you tested it?

File: code/modules/overmap/ships/engines/thermal.dm [hunk 1]
```diff
@@ -98,7 +98,7 @@
 	if(!removed)
 		return 0
 	. = calculate_thrust(removed)
-	playsound(loc, 'sound/machines/thruster.ogg', 100 * thrust_limit, 0, world.view * 4, 0.1)
+	playsound(loc, 'sound/machines/thruster.ogg', 300 * thrust_limit, 0, world.view * 4, 0.1)
```
Comment (Ghommie) [inline code/modules/overmap/ships/engines/thermal.dm]: Implying the rocket noozle is not located INSIDE the spaceship, I'd say the real issue here is how the playsound() use_pressure arg is TRUE, which can make the sound unaudible at a distance longer than contact range. Surprised it wasn't fixed before.

```suggestion
	playsound(loc, 'sound/machines/thruster.ogg', 100 * thrust_limit, 0, world.view * 4, 0.1, FALSE, FALSE)
```

**Comment by comma** [comment]
I'm pretty sure that BYOND caps volume of sound at 100 so just adding value there won't do shit.
The trick to make them hearable is to adjust playsound argument that adds extra distance, aka distance at which sound will not start to fall off yet.

**Comment by Ghommie** [comment]
The playsound extrarange arg is already set to `world.view * 4`, and its falloff of 0.1 (vs 0.5 standard).
As I said before, all of this means nothing if the source turfs has no pressure at all.

**Pull request closed without merging**